utils/build_video_reid_dataset.py

```diff
@@ -0,0 +1,190 @@
+import os
+import pickle
+import random
+import shutil
+import warnings
+
+import pandas as pd
+
+from DataProcessing.DB.dal import get_entries, Crop, create_session
+from DataProcessing.dataProcessingConstants import NAME_TO_ID
+from DataProcessing.utils import im_name_format
+
+BASE_CROPS_LOCATION = '/mnt/raid1/home/bar_cohen/'
+# DATASET_OUTPUT_LOCATION = '/mnt/raid1/home/bar_cohen/OUR_DATASETS/DukeMTMC-VideoReID' # keep this, it is for video location
+DATASET_OUTPUT_LOCATION = '/home/bar_cohen/KinderGuardian/fast-reid/datasets/same_day_0308' # this is for image
+
+
+def im_name_in_mars(crop: Crop, track_counter, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}C{crop.cam_id}T{track_counter:04d}F{crop_id:03d}.png'
+
+def im_name_in_duke(crop: Crop, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}_C{crop.cam_id}_F{crop_id + 1:04d}_X{crop.frame_num:05d}.png'
+
+def im_name_in_img_duke(crop:Crop, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}_c{crop.cam_id}_f{crop_id:07d}.jpg'
+
+def convert_to_img_reid_duke_naming(dataset_path:str, query_days, same_day=False):
```

Comment:
we have the file `utils/build_reid_dataset.py` no?
If so, why do we have here code for creating img-based dataset?

Reply:
that file is no longer relevant for building an image based dataset from our DB